lib/active_admin/qiniu_input/extend.rb

```diff
@@ -10,8 +10,9 @@ def qiniu_video_column(attribute)
     column attribute do |model|
       url = model.try(attribute)
       url.present? ?
-        video_tag(url, width: 150, controls: true, preload: 'auto', controlsList: 'nodownload') :
-        url
+        (video(width: 150, controls: true, preload: 'auto', controlsList: 'nodownload') do
+          source src: url
+        end) : url
```

Comment:
when url is nil, return nil value using `url.present?  ? xx : url` is not making sense,
use 
```
if url.present?
  xxx
end
```
to make your codes readable

Reply:
return nil is acceptable, ActiveAdmin will display nothing